pxf/pxf-service/src/main/java/org/apache/hawq/pxf/service/utilities/ProtocolData.java

```diff
@@ -95,6 +95,7 @@ public ProtocolData(Map<String, String> paramsMap) {
         accessor = getProperty("ACCESSOR");
         resolver = getProperty("RESOLVER");
         fragmenter = getOptionalProperty("FRAGMENTER");
+        metadata = getOptionalProperty("METADATA");
```

Comment:
This is used to when the getMetdata api is invoked. This class helps extracting the metadata class from the profile xml